CHANGELOG.rst

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is inspired by `Keep a Changelog <https://keepachangelog.com/en/1.0.0/>`_
 and this project adheres to `Semantic Versioning <https://semver.org/spec/v2.0.0.html>`_.
 
+`v0.x.y`_ - Unreleased
```

Comment:
This breaks the documentation linter in the CI.
Unsure what version number should be next :)

I guess v0.11.3 according to the current versioning, even though v0.12.0 could be appropriate too :)

Reply:
I've fixed change log format